xml/System/Char.xml

```diff
@@ -4737,14 +4682,12 @@ This member is an explicit interface member implementation. It can be used only
 
  Use <xref:System.String.ToLower%2A?displayProperty=nameWithType> to convert a string to lowercase.
 
-
-
 ## Examples
  The following example demonstrates <xref:System.Char.ToLower%2A>.
```

Comment:
This also needs to be moved to the MemberGroup element

Reply:
done